docs/pages/example/visualize-population-density.html

```diff
@@ -0,0 +1,53 @@
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map', // container id
+    style: 'mapbox://styles/mapbox/streets-v9', // stylesheet location
```

Comment:
I have a personal vendetta against streets-v9 in examples. The flattened road labels are 🙅‍♂️